libs/discord.lua

```diff
@@ -0,0 +1 @@
+lib( "discord", { "libs/discord/discord.cpp" } )
```

Comment:
I more or less stole the layout from LivePP (source here, header in qcommon)

Reply:
Should be the other way around, third party code in libs, our code in source. in this case I would do client/discord.cpp

Reply:
Moved them.